doc/whats-new.md

```diff
@@ -1,157 +1,289 @@
+because `mean_hazard_curves` is set to `false`. Also, in that case, no
+output rows are incorrectly added to the database.
+
+There is a check on the path names used in the source model logic
+tree file: they must be relative paths, not absolute paths, to make it
+possible to port the input files across different machines and operating
+systems.
+
+There is a check on calculations starting from precomputed hazard results: if
+job.ini file contains a site model or logic tree files, the user is warned
+that they will not used for the subsequent calculation and should be removed.
+
+We improved the error message in the case of source models in format
+nrml/0.4 uncorrectly marked as nrml/0.5.
+
+The CSV rupture exporter now also exports the parameter
+`ses_per_logic_tree_path` in the header of the CSV file.
+
+A lot of work went into the .rst reports which are now more reliable than
+ever.
+
+oq commands
+-----------
+
+There is a new command `oq shell` that starts an IPython shell (or Python
+shell, if IPython is not available) in the engine virtual environment.
+This is very useful to explore interactively the results of a computation.
+
+There is a new command `oq abort <job_id>` that allows to kill
+properly running calculations.
+
+The command `oq engine
+--delete-calculation <job_id>` has been extended to abort running calculations
+before removing them from the database and the file system.
+
+The command `oq zip` has been extended and now works for all kind of
+calculations.
+
+The command `oq extract` has now a signature consistent with `oq export`
+and has grown the ability to connect to a remote engine server.
+
+The `oq dump` command has been extended so that it is possible to extract
+a single calculation, if so desired.
+
+IT
+---
+
+We have specialized packages for clusters. The packages meant for the controller
+node includes rabbitmq and celery, which are missing in the packages meant for
+the worker nodes. Previously, we were installing rabbitmq and celery even when not
+needed.
+
+It is possible to specify a `custom_tmp` variable in the openquake.cfg file
+to configure a custom path for the temporary directory. This is useful in
+situations when the system /tmp directory is in a partition which is
+too small (this happened to users with a machine in the Azure cloud).
+
+The version of numpy officially supported by the engine is 1.11 however
+we made some work to ensure that the tests run also with the most recent
+versions. In the near future we will probably migrate to numpy 1.14.
+
+Since we added a field with the process ID in the job table in order to
+implement the abort functionality, upgrading to engine 2.9 requires a
+database migration if you have an older version of the engine and you want
+to keep your old calculations. You will just need to say yes the first time you
+run a calculation, when the engine will ask you to upgrade the database.
+The migration is immediate and safe.
```

Comment:
I would add "Docker containers can be used now to deploy a multi-node installation of the Engine on a cluster".

Reply:
Done